libraries/file/src/test/java/com/paritytrading/parity/file/taq/TAQWriterTest.java

```diff
@@ -9,99 +9,101 @@
 
     @Test
     public void writeWithDefaultConfiguration() throws Exception {
-	TAQ.Quote quote = new TAQ.Quote();
-
-	quote.date            = "2016-01-01";
-	quote.timestampMillis = 8 * 60 * 60 * 1000;
-	quote.instrument      = "FOO";
-	quote.bidPrice        = 100.50;
-	quote.bidSize         = 1000;
-	quote.askPrice        = 100.75;
-	quote.askSize         = 250;
-
-	ByteArrayOutputStream out = new ByteArrayOutputStream();
-
-	TAQWriter writer = new TAQWriter(out);
-
-	writer.write(quote);
-	writer.flush();
-
-	String output = "" +
-	    "Date\t" +
-	    "Timestamp\t" +
-	    "Instrument\t" +
-	    "Record Type\t" +
-	    "Bid Price\t" +
-	    "Bid Size\t" +
-	    "Ask Price\t" +
-	    "Ask Size\t" +
-	    "Trade Price\t" +
-	    "Trade Size\t" +
-	    "Trade Side\n" +
-	    "2016-01-01\t" +
-	    "08:00:00.000\t" +
-	    "FOO\t" +
-	    "Q\t" +
-	    "100.50\t" +
-	    "1000\t" +
-	    "100.75\t" +
-	    "250\t" +
-	    "\t" +
-	    "\t" +
-	    "\n";
-
-	assertEquals(output, out.toString("US-ASCII"));
+        TAQ.Quote quote = new TAQ.Quote();
+
+        quote.date            = "2016-01-01";
+        quote.timestampMillis = 8 * 60 * 60 * 1000;
+        quote.instrument      = "FOO";
+        quote.bidPrice        = 100.50;
+        quote.bidSize         = 1000;
+        quote.askPrice        = 100.75;
+        quote.askSize         = 250;
+
+        try (
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            TAQWriter writer = new TAQWriter(out);) {
```

Comment:
I would prefer to have this formatted as follows:
```java
try (ByteArrayOutputStream out = new ByteArrayOutputStream();
        TAQWriter writer = new TAQWriter(out)) {
    writer.write(quote);
    writer.flush();

    ...
```

What do you think?

Reply:
👍